src/main.py

```diff
@@ -12,6 +12,7 @@
 def welcome():
 	return "Fund Management Service"
 
+
 @app.route('/loan/add', methods = ['POST'])
```

Comment:
route name can be improved. 
Just `/loan` would be good. The method body will tell what the action will be.